x/market/handler.go

```diff
@@ -23,6 +24,29 @@ func NewHandler(k Keeper) sdk.Handler {
 	}
 }
 
+// Returns true if the amount of total coins swapped INTO the denom asset during a 24 hr window
+// (block approximation) exceeds 1% of coinissuance.
+func exceedsDailySwapLimit(ctx sdk.Context, k Keeper, swapCoin sdk.Coin) bool {
+	curDay := ctx.BlockHeight() / util.BlocksPerDay
+
+	// Disable trades on the first day
+	if curDay != 0 {
+		curIssuance := k.mk.GetIssuance(ctx, swapCoin.Denom, sdk.NewInt(curDay))
+		prevIssuance := k.mk.GetIssuance(ctx, swapCoin.Denom, sdk.NewInt(curDay-1))
+
+		if curIssuance.GT(prevIssuance) {
+			dailyDelta := curIssuance.Sub(prevIssuance).Add(swapCoin.Amount)
+
+			// If daily inflation is greater than 1%
+			if dailyDelta.MulRaw(100).GT(curIssuance) {
+				return true
+			}
+		}
+	}
+
+	return false
```

Comment:
To disable trades on the first day, I think it should return "true"

Reply:
@YunSuk-Yeo just inverted the comment to be consistent with the code. On the first day we want to allow all trades because 1) there won't be enough Terra to do damage anyway, and 2) we need to build up issuance history.